js/form.js

```diff
@@ -0,0 +1,35 @@
+const adForm = document.querySelector('.ad-form');
+const adFormElements = document.querySelectorAll('.ad-form__element');
+const mapFiltersForm = document.querySelector('.map__filters');
+const mapFiltersFormElements = mapFiltersForm.querySelectorAll('.map__filter');
+const mapFiltersFormFeatures = mapFiltersForm.querySelector('.map__features');
+
+const diactivateForm = () => {
+  adForm.classList.add('ad-form--disabled');
+  mapFiltersForm.classList.add('ad-form--disabled');
+  mapFiltersFormFeatures.setAttribute('disabled', 'disabled');
```

Comment:
Атрибут `disabled` присутствует без какого либо значения поэтому лучше:
```suggestion
  mapFiltersFormFeatures.setAttribute('disabled', '');
```